external/storm-kafka/src/jvm/org/apache/storm/kafka/ZkCoordinator.java

```diff
@@ -80,22 +81,24 @@ public void refresh() {
             List<Partition> mine = KafkaUtils.calculatePartitionsForTask(brokerInfo, _totalTasks, _taskIndex);
 
             Set<Partition> curr = _managers.keySet();
-            Set<Partition> newPartitions = new HashSet<Partition>(mine);
+            Set<Partition> newPartitions = new HashSet<>(mine);
             newPartitions.removeAll(curr);
 
-            Set<Partition> deletedPartitions = new HashSet<Partition>(curr);
+            Set<Partition> deletedPartitions = new HashSet<>(curr);
             deletedPartitions.removeAll(mine);
 
-            LOG.info(taskId(_taskIndex, _totalTasks) + "Deleted partition managers: " + deletedPartitions.toString());
+            LOG.info("{}Deleted partition managers: {}",
```

Comment:
Missing space

Reply:
This space is added in ``taskId`` method.
``return "Task [" + (taskIndex + 1) + "/" + totalTasks + "] ";``

Reply:
Ok, thanks.